client/modules/IDE/components/EditableInput.stories.jsx

```diff
@@ -0,0 +1,13 @@
+import EditableInput from './EditableInput';
+
+export default {
+  title: 'IDE/EditableInput',
+  component: EditableInput
+};
+
+export const Default = {
+  args: {
+    value: 'an@email.com',
+    label: 'Example'
+  }
+};
```

Comment:
We aren't seeing the pencil icon here due to an issue with `svgr`.  I think it's removing the `viewbox`.  Just FYI, we can fix that later.